src/main/resources/templates/index.html

```diff
@@ -0,0 +1,215 @@
+  <![endif]-->
+   
+    <!--====== PRELOADER PART START ======-->
+
+    <div class="preloader">
+        <div class="loader">
+            <div class="ytp-spinner">
+                <div class="ytp-spinner-container">
+                    <div class="ytp-spinner-rotator">
+                        <div class="ytp-spinner-left">
+                            <div class="ytp-spinner-circle"></div>
+                        </div>
+                        <div class="ytp-spinner-right">
+                            <div class="ytp-spinner-circle"></div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!--====== PRELOADER PART ENDS ======-->
+    
+    <!--====== NAVBAR TWO PART START ======-->
+
+    <section class="navbar-area">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12">
+                    <nav class="navbar navbar-expand-lg">
+
+                        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarTwo" aria-controls="navbarTwo" aria-expanded="false" aria-label="Toggle navigation">
+                            <span class="toggler-icon"></span>
+                            <span class="toggler-icon"></span>
+                            <span class="toggler-icon"></span>
+                        </button>
+
+                        <div class="collapse navbar-collapse sub-menu-bar" id="navbarTwo">
+                            <ul class="navbar-nav m-auto">
+                                <li class="nav-item active"><a class="page-scroll" href="#home">start</a></li>
+                                <li class="nav-item active"><a class="page-scroll" href="#confirmed-graph">Potwierdzone Przypadki</a></li>
+                                <li class="nav-item active"><a class="page-scroll" href="#deaths-section">Zgony</a></li>
+                            </ul>
+                        </div>
+                    </nav> <!-- navbar -->
+                </div>
+            </div> <!-- row -->
+        </div> <!-- container -->
+    </section>
+
+    <!--====== NAVBAR TWO PART ENDS ======-->
+    
+    <!--====== HOME PART START ======-->
+
+    <section id="home">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-4">
+                    <div class="home-content">
+                        <h1 class="title" id="current-deaths">0</h1>
+                        <p class="text">Aktualna liczba zgonów w Polsce</p>
+                    </div>
+                </div>
+                <div class="col-lg-4">
+                    <div class="home-content">
+                        <h1 class="title" id="current-confirmed">0</h1>
+                        <p class="text">Aktualna liczba zachorowań w Polsce</p>
```

Comment:
Aktualna liczba zachorowań że w tym momencie tyle choruje czy że do tej pory łącznie tyle chorowało?

Reply:
Ew. możesz zmienić na "Aktualnie chorujący" jeśli dotyczy to tylko osób w tym momencie (a nie wszystkich dotychczasowych przypadków z których część już się skończyła - wyzdrowieniem lub zgonem).

Reply:
Aktualna liczba potwierdzonych przypadków w Polsce?